sass/_omni-search.scss

```diff
@@ -0,0 +1,219 @@
+@import "general-mixins";
+@import "media-queries";
+$form-padding: 0.5rem;
+
+
+@mixin omnisearch-style($version) {
+  @if $version == style-1 {
+    @content; // calls the @mga version here...
```

Comment:
I meant that both statements return `@content` but it's doing different checks for `$version`. So how does it know that it's returning the correct styled version?